internal/terraform/resource.go

```diff
@@ -28,9 +28,29 @@ type Resource struct {
 	Version        types.String `json:"version" toml:"version" xml:"version" yaml:"version"`
 }
 
-// FullType returns full name of the type of the resource, including the provider name
-func (r *Resource) FullType() string {
-	return r.ProviderName + "_" + r.Type
+/*
+Spec Returns the resource spec addresses a specific resource in the config.
```

Comment:
First word after function name should start with lowercase. Also generally multi-lines comment is preferred to still be with `//`